pasta/base/annotate.py

```diff
@@ -1490,11 +1490,15 @@ def check_slice_includes_step(self, node):
     def visit_FormattedValue(self, node):
       self.visit(node.value)
       if node.conversion != -1:
-        self.attr(
-            node,
-            'conversion', [self.ws, '!', chr(node.conversion)],
-            deps=('conversion',),
-            default='!%c' % node.conversion)
+        # If we have a self-documenting token, eat the '='.
```

Comment:
You could add this link in a comment for helpful documentation

https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging